src/main/java/frc/controllers/Axis.java

```diff
@@ -22,9 +22,12 @@ public Axis(Joystick joystick, int axis, boolean invert) {
 	 * Add Deadband to remove false movement
 	 * @return value of specified axis
 	 */
-	public double get() {
-		//TODO: get axis
-		return 0;
+	public double get() 
+	{
+		if (invert == true) 
+			return -axis;
```

Comment:
The axis parameter represents the axis number of get from the joystick.
Look into joystick.getRawAxis();
https://first.wpi.edu/FRC/roborio/beta/docs/java/edu/wpi/first/wpilibj/GenericHID.html#getRawAxis-int-